src/Common/src/TypeSystem/Common/TargetDetails.cs

```diff
@@ -33,6 +33,22 @@ public enum TargetOS
     }
 
     /// <summary>
+    /// Specifies 
+    /// </summary>
+    public enum TargetApplicationModel
+    {
+        Unknown,
+        /// <summary>
+        /// Cross-platform console model
+        /// </summary>
+        CoreRT,
+        /// <summary>
+        /// Windows-specific UWP model
+        /// </summary>
+        NetNative
```

Comment:
(https://github.com/dotnet/corert/blob/master/Documentation/intro-to-corert.md describes how we call different things)